ll-list-control.html

```diff
@@ -117,34 +129,30 @@
       // Use this to clean up anything you did in `attached`.
     },
 
-    // Element Behavior
-
-    /**
-     * The `ll-list-control-lasers` event is fired whenever `fireLasers` is called.
-     *
-     * @event ll-list-control-lasers
-     * @detail {{sound: String}}
-     */
-
-    /**
-     * Sometimes it's just nice to say hi.
-     *
-     * @param {string} greeting A positive greeting.
-     * @return {string} The full greeting.
-     */
-    sayHello: function(greeting) {
-      var response = greeting || 'Hello World!';
-      return 'll-list-control says, ' + response;
+    _showEmptyText: function(){
+      if(this.items && this.items.length > 0){
+        this.$.listEmpty.style.display = "none";
+      }
+      else{
+        this.$.listEmpty.style.display = "block";
+      }
     },
-
     /**
-     * Attempt to destroy this element's enemies with a beam of light!
-     *
-     * Or, at least, dispatch an event in the vain hope that someone else will
-     * do the zapping.
+     * Sorts the initial list of returned images by the image sortOrder property.
+     * @private
      */
-    fireLasers: function() {
-      this.fire('ll-list-control-lasers', {sound: 'Pew pew!'});
+    _prepareForRemoval: function(e){
+      this.selectedItemId = e.target.id;
+      this.$.removeItemButton.style.display = "block";
+    },
+    _fireRemoveSelected: function(){
+      this.fire('remove-new', {selectedItemId: this.selectedItemId});
```

Comment:
For any events, we want to update the JSDocs comments. This way the docs will be auto updated to show all of the possible events to subscribe to.